src/components/blocksmith-field/components/youtube-video-block-modal/Preview.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+import YouTube from 'react-youtube';
+import classNames from 'classnames';
+import { Media } from 'reactstrap';
+import { BackgroundImage } from 'components';
+import damUrl from 'plugins/dam/damUrl';
+import './Preview.scss';
+
+export default function Preview(props) {
+  const { formState, width = 510, className, imageNode } = props;
+  const { autoplay, start_at: startAt, id, poster_image_ref: posterImageRef } = formState.values;
+
+  const [isPlayerReady, setIsPlayerReady] = useState(false);
+  const [showVideo, setShowVideo] = useState(false);
+  const [player, setPlayer] = useState(null);
+
+  const handleOnReady = (event) => {
+    const player = event.target;
+    setIsPlayerReady(true);
+    setPlayer(player);
+  }
+
+  const shouldAutoPlay = (imageNode && showVideo) || autoplay;
+
+  if (player) {
+    if (shouldAutoPlay) {
+      player.seekTo(startAt);
+      // check if player is in "PAUSED" state ( https://developers.google.com/youtube/iframe_api_reference )
+      if (player.getPlayerState() === 2) {
```

Comment:
@rsumilang is it possible to use `YT.PlayerState.PAUSED` instead of 2? i find it easier to read.